core/ansi_escape_parser.js

```diff
@@ -37,6 +37,10 @@ function ANSIEscapeParser(options) {
     this.mciReplaceChar = miscUtil.valueWithDefault(options.mciReplaceChar, '');
     this.termHeight = miscUtil.valueWithDefault(options.termHeight, 25);
     this.termWidth = miscUtil.valueWithDefault(options.termWidth, 80);
+    this.breakWidth = this.termWidth;
+    if(!(_.isNil(options.artWidth)) && options.artWidth > 0 && options.artWidth < this.breakWidth) {
```

Comment:
We should probably use `parseInt()` and `isNaN()` here on `artWidth`.

Reply:
I had done that in art.js but your right, shouldn't leave it to that. Updated.